contrib/puppetlabs-lumogon/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change log
+
+All notable changes to this project will be documented in this file.
+
+## 0.4.0 (June 8, 2017)
```

Comment:
What was the rationale of the 0.4.0 version? Would you see us shipping a new version for every Lumogon release? At which point should this actually pin to that release rather than use latest? What about versions of Lumogon that don't contain new capabilities?